packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -239,11 +250,22 @@ const UsersProfileFn = ({terms, slug, classes}: {
         <AnalyticsContext pageContext={"userPage"}>
           {/* Bio Section */}
           <SingleColumnSection>
-            <div className={classes.usernameTitle}>
-              {username}
+            <div className={classes.title}>
```

Comment:
is this title wrapper still relevant or is that a leftover from a previous iteration of the design?